react/features/toolbox/components/web/Toolbox.js

```diff
@@ -41,6 +41,9 @@ import {
     SettingsButton,
     openSettingsDialog
 } from '../../../settings';
+import {
+    VideoBlurButton
+} from '../../../blur';
```

Comment:
style: the imports should be sorted alphabetically first by the filename. Also this import can be done on a single line.

Reply:
Reordered imports

Reply:
Alphabetically '../../../blur' should go after ''../../../base/*', isn't it?